CMakeLists.txt

```diff
@@ -608,6 +608,16 @@ if(NOT IOS AND CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND CMAKE_OSX_ARCHITECTURES M
   set(CAFFE2_CUSTOM_PROTOC_EXECUTABLE "${PROJECT_SOURCE_DIR}/build_host_protoc/bin/protoc")
 endif()
 
+# Cross compile setup targeting Jetson platforms:
+#  Use pre-built protobuf supplied on cmake command line
+if(CMAKE_CROSSCOMPILING AND USE_CUDA)
```

Comment:
I'm not sure if we want to remove the extra condition. [`CMAKE_CROSSCOMPILING` will be set `whenever the CMAKE_SYSTEM_NAME variable has been set manually ](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html). I'm not sure if we want to assume it's really for cross compiling. I'd really appreciate any suggestions here. 

Another consideration is that the change was only tested targeting Jetson and we do have other setups (with CROSS_COMPILING=TRUE) that can overlap/conflict with this PR. An example is the protoc setup above this line for OSX. It's unclear to me how much we'd want to share the setup since the environment can be quite different and the future changes may also diverge.

Reply:
Actually I think now `sleef` and `protoc` setup are not necessary in the cmake setup. Providing the variables from command line seems enough. I just removed them from this PR for now.